### ISSUE TITLE
--user option demotes watchdog resulting in setOomScore Permission denied error

### ISSUE THREAD

**Issue by majgis** [phusion/passenger#1858]
**Setup:**
Centos 6.8
SELinux Disabled
Phusion Passenger Standalone v5.0.29

```
passenger start \
  --max-pool-size ${value1} \
  --min-instances ${value2} \
  --app-type node \
  --startup-file ${value3} \
  --environment ${value4} \
  --port ${value5} \
  --daemonize \
  --user ${APP_USER} \
  --static-files-dir ${value7} \
  --log-file ${value8} \
  --nginx-config-template ${value9}
```

With this configuration, the watchdog process is run as ${APP_USER} and results in this error getting repeatedly logged:

```
[ 2016-08-03 23:24:21.8964 17662/7f503b512700 age/Wat/WatchdogMain.cpp:221 ]: setOomScore(0, 1) failed due to error: Permission denied
```

Removing the --user option allows the watchdog to run as root, but the application runs as root as well.

**Comment by majgis** [comment]
What are the implications of running watchdog as the application user?   Beyond this log message, what other consequences can we expect?

**Comment by CamJN** [comment]
The Watchdog should perform any tasks that require root before dropping privileges, so this is a bug, but there shouldn't be any other implications.

**Comment by mitomm** [comment]
I'm still getting `[ 2016-11-21 14:38:34.0239 10085/7f06141da700 age/Wat/WatchdogMain.cpp:221 ]: setOomScore(0, 1) failed due to error: Permission denied` in the passenger log once every hour.

I'm using Passenger version 5.0.30.

Please let me know if you need further details.

**Comment by CamJN** [comment]
This should be fixed in the upcoming 5.1.0 release.

**Comment by mitomm** [comment]
Oh right, sorry, I didn't notice the 5.1.0 tag and thought I has already been fixed in 5.0.30. 

Any plans on when version 5.1 should be released?

**Comment by CamJN** [comment]
It should be released in the next couple days barring any last minute surprises.
